rdmd.d

```diff
@@ -839,6 +835,102 @@ import std.stdio, std.algorithm, std.array, std.ascii, std.base64,
     std.zlib;
 ";
 
+/**
+    Joins together the code provided via an `--eval`
```

Comment:
I believe the agreed upon convention is to have the ddoc content aligned with the comment, which in this case means no leading whitespace.

Reply:
Yes, please follow the [DStyle](https://dlang.org/dstyle.html#phobos_documentation) for new code.